lib/adapters/MochaAdapter.js

```diff
@@ -52,10 +52,25 @@ export default class MochaAdapter extends EventEmitter {
     // a "test start".
     if (mochaTest.errors !== undefined) {
       var status = (mochaTest.state === undefined) ? 'skipped' : mochaTest.state
-
-      // Test end.
+      let errors = []
+
+      // Mocha has indeed only one error per test, but by this is avoided the
```

Comment:
There's something wrong with the comment here. "but by this is"? Not sure what you're trying to say.